src/pages/workflows/common/WDLEditor.tsx

```diff
@@ -0,0 +1,178 @@
+import Editor from '@monaco-editor/react';
+import React from 'react';
+
+/* The syntax highlighting is not exactly what is used to be with the addition
+ * of the new editor. This ticket is a follow-up to update the syntax highlighting
+ * to better resemble what we had using PrismJs: https://broadworkbench.atlassian.net/browse/WX-1838
+ */
```

Comment:
Another thing that I _think_ is true: Monaco can use syntax highlighting from VS Code, and there are community-created highlighters for WDL

Reply:
I should clarify – maybe add that fact to the comment, don't necessarily need to implement it immediately.